lib/utils.js

```diff
@@ -669,3 +671,64 @@ function autoEnd (stream, eventName) {
     stream.flushSync()
   }
 }
+
+/**
+ * Returns a object which will be used to handle custom levels
+ *
+ * @param {string|object} cLevels Custom Levels can be object or string
+ *
+ * @returns {object} A object like level: level_name
+ */
+function handleCustomlevelsOpts (cLevels) {
```

Comment:
Thank you for providing jsdoc for the new function. However, the documentation should provide some real value over what the function signature already provides:

```suggestion
/**
 * Parse a CSV string or options object that specifies
 * configuration for custom levels.
 *
 * @param {string|object} cLevels An object mapping standard level
 * names to new level names, e.g. `{ info: 'information') }`, or a
 * CSV string in the format `level:new_name`, e.g.
 * `info:information,debug:devel`.
 *
 * @returns {object} A object mapping known level names to customized
 * level names.
 */
function handleCustomlevelsOpts (cLevels) {
```

Reply:
Will be done.